lib/warden/jwt_auth/strategy.rb

```diff
@@ -7,7 +7,11 @@ module JWTAuth
     # Warden strategy to authenticate an user through a JWT token in the
     # `Authorization` request header
     class Strategy < Warden::Strategies::Base
+      include JWTAuth::Import['dispatch_requests']
+
       def valid?
+        return false if path_is_dispatch_request_path?
+
         token_exists? && issuer_claim_valid?
```

Comment:
```suggestion
  token_exists? && issuer_claim_valid? && !path_is_dispatch_request_path?
```
We can keep it as a one-liner. WDYT?